src/components/Question.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import parse from "html-react-parser";
+import styled from "styled-components";
+import Radio from "@material-ui/core/Radio";
+import AnswerBox from "./AnswerBox";
+import { withStyles } from "@material-ui/core/styles";
+import { green, orange, red } from "@material-ui/core/colors";
+
+const CorrectRadio = withStyles({
+  root: {
+    color: green[600],
+    "&$checked": {
+      color: green[600],
+    },
+  },
+  checked: {},
+})((props) => <Radio color="default" {...props} />);
+
+const WrongRadio = withStyles({
+  root: {
+    color: orange[800],
+    "&$checked": {
+      color: orange[800],
+    },
+  },
+  checked: {},
+})((props) => <Radio color="default" {...props} />);
+
+const Container = styled.div`
+  padding: 1rem;
+  max-width: 768px;
+`;
+const Title = styled.h2`
+  margin: 0px;
+`;
+
+const QuestionForm = styled.div`
+  margin-top: 1.5rem;
+`;
+const AlternativesBox = styled.div`
+  padding: 0.5rem 0rem;
+  background-color: white;
+`;
+
+const RadioLabel = styled.span`
```

Comment:
Pensando na experiência do usuário, acho que seria melhor esse elemento ser uma label, ligada ao radio button pelo atributo `for`. Assim a pessoa pode clicar em qualquer lugar da linha e não só no input.